pcsc/src/lib.rs

```diff
@@ -571,7 +571,7 @@ impl Context {
         scope: Scope,
     ) -> Result<Context, Error> {
         unsafe {
-            let mut ctx: ffi::SCARDCONTEXT = DUMMY_LONG;
+            let mut ctx: ffi::SCARDCONTEXT = DUMMY_LONG as ffi::SCARDCONTEXT;
```

Comment:
I'm wondering. Why don't you want to use `mem::uninitialized` here ?
As we do not read it and `SCARDCONTEXT` does not implement `Drop`, there shouldn't be any undefined behaviour here, right ?
There would be no need for casting type then.

Please, clarify if I'm wrong.

Reply:
`uninitialized` is unsafe. Of course we are inside an `unsafe` block, but if it can be easily avoided, I think it's better.

See ac9e45e66e1716447b9d0c4f526a37d9f75f9cfd where I've made this change.

Reply:
Well, it could be safer without `uninitialized`, was just wondering ^^